ocfweb/docs/docs/staff/tips/twitch.md

```diff
@@ -0,0 +1,35 @@
+[[!meta title="Using Twitch and OBS"]]
+
+## Setting Up Your Stream
+
+In your settings on Twitch, please enable VODs (video on demand) especially if you're working on OCF stuff.
+To do this, go to https://dashboard.twitch.tv/settings/channel and enable "Store past broadcasts".
```

Comment:
IIRC this won't actually be a link on ocfweb. You need to use the `[]()` syntax to make it a link.

Reply:
I tested it with the original document, and ocfweb was still able to parse the link even without formatting it through Markdown. I've converted the links anyway though, but I found that interesting.

Reply:
Oh neat, okay. Not sure where I got that impression then.